addons/website_slides/tests/test_slide_slide.py

```diff
@@ -8,6 +8,41 @@
 from odoo.tools import mute_logger
 
 
+class TestSlidesSlideMail(slides_common.SlidesCase):
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.test_template_slides = cls.env['mail.template'].with_user(cls.user_manager).create({
+            'auto_delete': True,
+            'body_html': '<p>Fist fight with <t t-out="object.user_id.name"/></p>',
+            'email_from': '{{ object.user_id.email_formatted or user.email_formatted or "" }}',
+            'model_id': cls.env['ir.model']._get_id('slide.slide'),
+            'name': 'Test Slide Template',
+            'subject': 'Test {{ object.name }}'
+        })
+        cls.slide.write({
+            'partner_ids': [(6, 0, [cls.customer.id, cls.user_emp.partner_id.id, cls.user_portal.partner_id.id])]
```

Comment:
Detail: these trailing commas.

```suggestion
            'partner_ids': [(6, 0, [cls.customer.id, cls.user_emp.partner_id.id, cls.user_portal.partner_id.id])],
```